nautobot/utilities/testing/views.py

```diff
@@ -416,9 +417,18 @@ def test_create_object_with_permission(self):
             self.assertHttpStatus(self.client.post(**request), 302)
             self.assertEqual(initial_count + 1, self._get_queryset().count())
             if hasattr(self.model, "last_updated"):
-                self.assertInstanceEqual(self._get_queryset().order_by("last_updated").last(), self.form_data)
+                instance = self._get_queryset().order_by("last_updated").last()
+                self.assertInstanceEqual(instance, self.form_data)
             else:
-                self.assertInstanceEqual(self._get_queryset().last(), self.form_data)
+                instance = self._get_queryset().last()
+                self.assertInstanceEqual(instance, self.form_data)
+
+            # Verify ObjectChange creation
+            objectchanges = ObjectChange.objects.filter(
+                changed_object_type=ContentType.objects.get_for_model(instance), changed_object_id=instance.pk
+            )
+            self.assertEqual(len(objectchanges), 1)
+            self.assertEqual(objectchanges[0].action, ObjectChangeActionChoices.ACTION_CREATE)
```

Comment:
Would this make this test valid for non-changelogged models as well?
```suggestion
            if hasattr(self.model, "to_objectchange"):
                # Verify ObjectChange creation
                objectchanges = ObjectChange.objects.filter(
                    changed_object_type=ContentType.objects.get_for_model(instance),
                    changed_object_id=instance.pk,
                )
                self.assertEqual(len(objectchanges), 1)
                self.assertEqual(objectchanges[0].action, ObjectChangeActionChoices.ACTION_CREATE)
```